resolwe_bio/processes/import_data/annotation.yml

```diff
@@ -65,14 +65,15 @@
   category: upload
   persistence: RAW
   description: >
-    Upload annotation in GTF format.
+    The GTF format is identical to GFF version 2 so this process
```

Comment:
Wouldn't it be more descriptive to also preserve the original description (i.e. "Upload annotation in GTF format.") and add the new text as an addition?